### PR TITLE
add db tables for product attribute variant

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: htdocs/install/mysql/tables/llx_product_attribute_extrafields.key.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+-- ===================================================================
```
Comment (eldy) [inline htdocs/install/mysql/tables/llx_product_attribute_extrafields.key.sql:1]: I suggest to rename the table into
llx_product_attribute_extrafields so the table will be create only if we activate the module variant.

Comment (frederic34) [reply htdocs/install/mysql/tables/llx_product_attribute_extrafields.key.sql:1]: rename the table or the file ?
llx_product_attribute_extrafields.sql to llx_product_attribute_extrafields-variants.sql

Comment (eldy) [reply htdocs/install/mysql/tables/llx_product_attribute_extrafields.key.sql:1]: > rename the table or the file ? llx_product_attribute_extrafields.sql to llx_product_attribute_extrafields-variants.sql

You're right, i mean rename the "file" only.

Comment (frederic34) [reply htdocs/install/mysql/tables/llx_product_attribute_extrafields.key.sql]: Does I need to remove migrations ?

**Pull request merged**